server/index.js

```diff
@@ -19,6 +21,16 @@ app.use((req, res, next) => {
       next(err);
 });
 
+app.get('/api/products', loginVerify, vendorVerify, async function (req, res, next) {
```

Comment:
is this the authentication process for only vendor? do we use it for user authorization?

Reply:
There are 3 auth middlewares in ``/middleware/auth.js``. ``loginVerify`` is for all accounts to check if they have signed in or not. ``userVerify`` is for USER identity, and ``vendorVerify`` is for VENDOR identity.